jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -353,6 +353,9 @@ void checkoutCode(String url, String sha) {
                                 [$class: 'SubmoduleOption', disableSubmodules: false, shallow: true, parentCredentials: true]]
         ]
     )
+    if (!fileExists("thirdparty/parquet-testing/data")) {
```

Comment:
This should be using something like `git submodule status`.  Could look for any line in the output that starts with `-`, as that indicates a submodule that has not been initialized.  The advantage of this approach is it works with any submodule (or more than one submodule) without needing to understand the layout of a specific submodule (and hope it doesn't change in a future commit).

Reply:
+1, `-` at the head if not been initialized

```
git submodule status
-d79a0101d90dfa3bbb10337626f57a3e8c4b5363 thirdparty/parquet-testing
```